include/CL/cl_exp_defined_builtin_kernels.h

```diff
@@ -88,8 +88,14 @@ typedef enum
   POCL_CDBI_MAGNITUDE_P512 = 35,
   POCL_CDBI_ORIENTED_NONMAX_P512 = 36,
   POCL_CDBI_GAUSSIAN3X3_P512 = 37,
-  POCL_CDBI_DBK_KHR_GEMM = 38,
-  POCL_CDBI_DBK_KHR_MATMUL = 39,
+  POCL_CDBI_DBK_EXP_GEMM = 38,
+  POCL_CDBI_DBK_EXP_MATMUL = 39,
+  /* see 'JPEG Defined Builtin Kernels:Usage' in
```

Comment:
```suggestion
  /* See 'JPEG Defined Builtin Kernels:Usage' in jpeg_dbk.rst for details on usage. */
```
Please use full sentences and more of the allowed 80 chars per line.